pages/home.md

```diff
@@ -1,31 +1,31 @@
 ---
 permalink: /
 layout: landing
-title: "U.S. Web Design Standards: A design system for the federal government"
+title: "U.S. Web Design System: A design system for the federal government"
 class: home
 hero:
   callout: A design system for the federal government
-  content: The Standards provide research-backed design patterns for building accessible, responsive, and consistent digital products for the federal government.
+  content: The Design System provides research-backed design patterns for building accessible, responsive, and consistent digital products for the federal government.
```

Comment:
As above, maybe this can be

**Design and build fast, accessible, mobile-friendly government websites backed by user research.**

Reply:
Updated in 060cb04.